misc/metrics/src/lib.rs

```diff
@@ -42,7 +43,8 @@ mod protocol_stack;
 mod relay;
 mod swarm;
 
-use prometheus_client::registry::Registry;
+pub use bandwidth::Transport as BandwidthMetricTransport;
```

Comment:
Maybe just call this `BandwidthTransport`? We are within the scope of `metrics` here so it is kind of redundant?